scout/adapter/mongo/case.py

```diff
@@ -892,11 +892,17 @@ def update_case_data_sharing(self, old_case: dict, new_case: dict):
             if key in old_case:
                 new_case[key] = old_case[key]
 
-    def _load_omics_variants(self, case_obj: dict, build: str, update: bool = False):
+    def _load_clinical_omics_variants(self, case_obj: dict, build: str, update: bool = False):
         """Load omics variants. The OMICS FILE type dict contains all we need to
         determine how to load variants (type, category etc)."""
 
-        for omics_file in OMICS_FILE_TYPE_MAP.keys():
+        CLINICAL_ORDERED_OMICS_FILE_TYPE_MAP = OrderedDict(
```

Comment:
This is a fix to the issue I'm describing [here](https://github.com/Clinical-Genomics/scout/pull/5236#issuecomment-2647581360)